docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -15,69 +15,53 @@ authenticate to external services.
 
 - (!docs/pages/includes/tctl.mdx!)
 - If you're new to SAML, consider reviewing our [SAML Identity Provider
-Reference](./saml-reference.mdx) before proceeding.
+  Reference](./saml-reference.mdx) before proceeding.
+- User with permission to create Service Provider resource. The preset `editor` role has this permission.
+- Test SAML application, also known as SAML Service Provider (SP). For this guide, we'll be using
+[samltest.id](https://samltest.id/) to create a test consumer of Teleport's SAML IdP.
 
-## Example external application
+## Step 1/3. Add samltest.id Service Provider to Teleport
```

Comment:
```suggestion
## Step 1/3. Add a Service Provider to Teleport
```
Using more general headings so users can create a mental model of the flow. Otherwise, I think this is too specific to the example we're using.

Reply:
Totally. I borrowed the copy from existing page but I felt the same way. updated this and your other suggestions in https://github.com/gravitational/teleport/pull/36467/commits/7193225bab9b229e414415717a268093a129f76f